### DISCUSSION TITLE
WASM: How can I save user default values?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#2263] [Ideas]
WASM: `class UserDefault` should 'save' also on web versions  of my app

@all
Maybe there is an easy way anywhere?

**Comment by paulocoutinhox** [comment]
Hi,

You use use my impl:
https://github.com/paulocoutinhox/axmol-native-helper

And to save locally the UserDefault im using it:
https://github.com/nativium/nativium-modules/tree/main/custom/modules/shared-data/implementation/cpp/nativium/data

Maybe help you.
